modules/AOR_Reports/aor_utils.php

```diff
@@ -206,7 +196,6 @@ function getConditionsAsParameters($report, $override = array())
         $disp = getDisplayForField($path, $condition->field, $report->report_module);
         $conditions[] = array(
             'id' => $condition->id,
-            'key' => $key,
```

Comment:
Does this cause any issues with parameters for dashlet reports? it reverts #4474.